Snowdog_Acm/templates/content/widget/sliders/list.phtml

```diff
@@ -0,0 +1,171 @@
+    $sliderAutoplaySpeed = $content->getData($sliderPrefix . 'speed');
+    $sliderPauseOnFocus = $content->getData($sliderPrefix . 'focus');
+    $sliderPauseOnHover = $content->getData($sliderPrefix . 'hover');
+    $sliderSlidesToShow = $content->getData($sliderPrefix . 'slidestoshow');
+    $sliderSlidesToScroll = $content->getData($sliderPrefix . 'slidestoscroll');
+    $sliderResponsiveConfig = $content->getData($sliderPrefix . 'responsive_cfg');
+
+    $sliderTitle = $block->escapeHtml($content->getData($sliderPrefix . 'title'));
+    $sliderDisplayTitle = $block->escapeHtml($content->getData($sliderPrefix . 'display_title'));
+    $sliderDisplayTitle = ($sliderDisplayTitle === 'no') ? false : true;
+    $sliderContentBefore =
+        $content->getData($sliderPrefix . 'ct_before') ?
+            $content->render($sliderPrefix . 'ct_before') :
+            false;
+    $sliderContentAfter =
+        $content->getData($sliderPrefix . 'ct_after') ?
+            $content->render($sliderPrefix . 'ct_after') :
+            false;
+
+    $sliderBanners = $content->getContentCollection($sliderPrefix . 'banners', $bannersAttributes);
+
+    foreach ($sliderBanners as $index => $banner) {
+        foreach ($pictureData as $pictureDataAttr) {
+            $bannersArray[$index][$pictureDataAttr] = $banner->getData($picturePrefix . $pictureDataAttr);
+        }
+
+        foreach ($pictureFiles as $pictureFile) {
+            $bannersArray[$index][$pictureFile] =
+                $banner->getData($picturePrefix . $pictureFile) ? $banner->getFile($picturePrefix . $pictureFile) : false;
+        }
+    }
+
+    // sorting banners collection by order
+    usort($bannersArray, function ($a, $b) {
+        return $a['order'] - $b['order'];
+    });
+    ?>
+
+    <?php
+    $sliderBlockBefore->setData([
+        'slider_html'       => 'before-slides',
+        'slider_class'      => $sliderClassName,
+        'wrapper_class'     => $sliderWrapperClass,
+        'display_title'     => $sliderDisplayTitle,
+        'slider_title'      => $sliderTitle,
+        'title_class'       => $sliderTitleClassName,
+        'content_before'    => $sliderContentBefore,
+        'infinite'          => $sliderInfinite,
+        'center_mode'       => $sliderCenterMode,
+        'dots'              => $sliderDots,
+        'arrows'            => $sliderArrows,
+        'autoplay'          => $sliderAutoplay,
+        'autoplay_speed'    => $sliderAutoplaySpeed,
+        'pause_on_focus'    => $sliderPauseOnFocus,
+        'pause_on_hover'    => $sliderPauseOnHover,
+        'slides_to_show'    => $sliderSlidesToShow,
+        'slides_to_scroll'  => $sliderSlidesToScroll,
+        'responsive_config' => $sliderResponsiveConfig
+    ]);
+    ?>
+    <?= $sliderBlockBefore->toHtml(); ?>
+
+    <?php if ($bannersArray): ?>
+        <?php foreach ($bannersArray as $banner): ?>
+            <?php
+            $bannerUrl = $block->escapeUrl($banner['url']);
```

Comment:
I think we could
- rename it to escapedBannerUrl 
- or assign just `$banner['url']` and use escapeUrl method where we have php echo statement.